src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -270,296 +281,27 @@ internal static void AddFunctionError(Dictionary<string, Collection<string>> fun
         }
 
         /// <summary>
-        /// Performs all required initialization on the host. Must be called before the host is started.
+        /// Performs all required initialization on the host.
+        /// Must be called before the host is started.
         /// </summary>
         public void Initialize()
```

Comment:
I've moved all of the code out of this method, into pipeline stage functions that gather relatively cohesive operations. This method has went from about 300 lines to 12 lines :)